concordia/static/js/asset-reservation.js

```diff
@@ -34,24 +57,45 @@ function attemptToReserveAsset(reservationURL) {
             window.setTimeout(attemptToReserveAsset, 60000, reservationURL);
         });
 
-    window.addEventListener('beforeunload', function() {
-        var payload = {
-            release: true,
-            csrfmiddlewaretoken: jQuery(
-                'input[name="csrfmiddlewaretoken"]'
-            ).val()
-        };
+    addUnloadListener(reservationURL);
+}
 
-        // We'll try Beacon since that's reliable but until we can drop support for IE11 we need a fallback:
-        if ('sendBeacon' in navigator) {
-            navigator.sendBeacon(
-                reservationURL,
-                new Blob([jQuery.param(payload)], {
-                    type: 'application/x-www-form-urlencoded'
-                })
-            );
-        } else {
-            jQuery.ajax({url: reservationURL, type: 'POST', data: payload});
-        }
-    });
+function checkForReviewers(reservationURL, findANewPageURL) {
```

Comment:
This is basically the same as the preceding function except for the message text. It seems like we should consolidate both of those and consider changing the assumption for transcription reservations to also receive a find-a-new-page URL since that's useful. This could either be generic wording (“Someone else is working on this page”) or refactoring this function to be entirely generic except for the branch which handles the HTTP 409 Conflict by using a callback to display the message rather than having it hard-coded here.

Reply:
I thought the transcription reservation popped up a modal. We don't want the review reservation to do that.

Reply:
It used to just be a banner message